### ISSUE TITLE
BUG: explore() invalid value encountered in double_scalars

### ISSUE THREAD

**Issue by rraymondgh** [geopandas/geopandas#2546]
- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the latest version of geopandas.

- [ ] (optional) I have confirmed this bug exists on the main branch of geopandas.

---

#### Code Sample, a copy-pastable example

```python
import geopandas as gpd

cities = gpd.read_file(gpd.datasets.get_path('naturalearth_cities'))
cities.assign(zero=0).explore("zero")
```

#### Problem description
If all values in `column` are zero results in warning:
```
mapclassify/classifiers.py:891: RuntimeWarning: invalid value encountered in double_scalars
  gadf = 1 - self.adcm / adam
```
Have not investigated where in **geopandas** `explore.py()` this is generated.  Reporting and will investigate further

#### Output of ``geopandas.show_versions()``

<details>

SYSTEM INFO
-----------
python     : 3.9.10 (main, Jan 15 2022, 11:48:00)  [Clang 13.0.0 (clang-1300.0.29.3)]
executable : /Users/rraymond/PycharmProjects/moviedb/venv/bin/python3.9
machine    : macOS-11.6.8-x86_64-i386-64bit

GEOS, GDAL, PROJ INFO
---------------------
GEOS       : 3.10.2
GEOS lib   : /usr/local/Cellar/geos/3.10.2/lib/libgeos_c.dylib
GDAL       : 3.4.1
GDAL data dir: /Users/rraymond/PycharmProjects/moviedb/venv/lib/python3.9/site-packages/fiona/gdal_data
PROJ       : 8.2.0
PROJ data dir: /Users/rraymond/PycharmProjects/moviedb/venv/lib/python3.9/site-packages/pyproj/proj_dir/share/proj

PYTHON DEPENDENCIES
-------------------
geopandas  : 0.11.1
numpy      : 1.23.2
pandas     : 1.4.3
pyproj     : 3.3.1
shapely    : 1.8.2
fiona      : 1.8.21
geoalchemy2: None
geopy      : 2.2.0
matplotlib : 3.5.3
mapclassify: 2.4.3
pygeos     : 0.12.0
pyogrio    : None
psycopg2   : None
pyarrow    : 9.0.0
rtree      : 1.0.0

</details>

**Comment by martinfleis** [comment]
That is just a warning, right? That is stemming from the fact that we are trying to create color bins using map classify and all your values are the same. We can probably check that case and skip the binning code.

**Comment by rraymondgh** [comment]
@martinfleis yes just a warning.  Surprised me when I saw it for first time, wasting time on looking for other data quality issues ...

**Comment by martinfleis** [comment]
It is most likely coming from this part. Which can be skipped for invariant arrays like the one above and `color` array can be created directly.

https://github.com/geopandas/geopandas/blob/1821b6d72791ce1f7da32db5b22e8803a3dc44b9/geopandas/explore.py#L448-L455

**Comment by martinfleis** [comment]
Fixed in https://github.com/pysal/mapclassify/pull/131
